synapse/storage/data_stores/main/schema/delta/56/signing_keys.sql

```diff
@@ -40,7 +40,7 @@ CREATE TABLE IF NOT EXISTS e2e_cross_signing_signatures (
     signature TEXT NOT NULL
 );
 
-CREATE UNIQUE INDEX e2e_cross_signing_signatures_idx ON e2e_cross_signing_signatures(user_id, target_user_id, target_device_id);
+CREATE INDEX e2e_cross_signing_signatures2_idx ON e2e_cross_signing_signatures(user_id, target_user_id, target_device_id);
```

Comment:
I'd either a) comment this out and add a note that this got replaced or b) just leave it and rely on it being cleaned up correctly